packages/ui/src/forum/components/threads/ThreadListItem.tsx

```diff
@@ -39,7 +39,7 @@ export const ThreadListItem = ({ thread, isArchive }: ThreadListItemProps) => {
         </Thread>
       </Tooltip>
 
-      <TextMedium bold>{thread.visiblePostsCount - 1}</TextMedium>
+      <TextMedium bold>{ thread.visiblePostsCount > 0 ? thread.visiblePostsCount - 1 : 0 }</TextMedium>
```

Comment:
```suggestion
      <TextMedium bold>{thread.visiblePostsCount > 0 ? thread.visiblePostsCount - 1 : 0}</TextMedium>
```

Reply:
But I don't think the individual linting issues should be fixed on this release.